<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scatter Plot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Band X</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QgsMapLayerComboBox" name="uiRaster1_">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <property name="allowEmptyLayer">
               <bool>true</bool>
              </property>
              <property name="excludedProviders">
               <stringlist/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QgsRasterBandComboBox" name="uiBand1_">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="currentIndex">
               <number>-1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Min</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="uiMin1_"/>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Max</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="uiMax1_"/>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Band Y</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QgsMapLayerComboBox" name="uiRaster2_">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <property name="allowEmptyLayer">
               <bool>true</bool>
              </property>
              <property name="excludedProviders">
               <stringlist/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QgsRasterBandComboBox" name="uiBand2_">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Min</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="uiMin2_"/>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Max</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="uiMax2_"/>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Mask (optional)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QgsMapLayerComboBox" name="uiMask_">
            <property name="currentIndex">
             <number>-1</number>
            </property>
            <property name="allowEmptyLayer">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Accuracy</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="uiAccuracy_">
            <item>
             <property name="text">
              <string>Estimated (faster)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Actual (slower)</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Bins</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="uiBins_">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Correlation analysis</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QCheckBox" name="uiShowMinMaxLine_">
              <property name="text">
               <string>Min-max line</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="showFittedLine_">
              <property name="text">
               <string>Fitted line</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="uiFit_">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="ImageView" name="uiImageView_" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QProgressBar" name="uiProgressBar_">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="uiExecute_">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/images/themes/default/propertyicons/action.svg</normaloff>:/images/themes/default/propertyicons/action.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>822</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsRasterBandComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsrasterbandcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>ImageView</class>
   <extends>QWidget</extends>
   <header>enmapboxapplications.scatterplotapp.core</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>uiRaster1_</sender>
   <signal>layerChanged(QgsMapLayer*)</signal>
   <receiver>uiBand1_</receiver>
   <slot>setLayer(QgsMapLayer*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uiRaster2_</sender>
   <signal>layerChanged(QgsMapLayer*)</signal>
   <receiver>uiBand2_</receiver>
   <slot>setLayer(QgsMapLayer*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uiRaster1_</sender>
   <signal>layerChanged(QgsMapLayer*)</signal>
   <receiver>uiMin1_</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>585</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uiRaster1_</sender>
   <signal>layerChanged(QgsMapLayer*)</signal>
   <receiver>uiMax1_</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>800</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uiRaster2_</sender>
   <signal>layerChanged(QgsMapLayer*)</signal>
   <receiver>uiMin2_</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>574</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uiRaster2_</sender>
   <signal>layerChanged(QgsMapLayer*)</signal>
   <receiver>uiMax2_</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>800</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uiBand1_</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>uiMin1_</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>502</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>644</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uiBand1_</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>uiMax1_</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>748</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uiBand2_</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>uiMin2_</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>482</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>640</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uiBand2_</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>uiMax2_</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>716</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
